src/test/java/seedu/address/commons/util/StringUtilTest.java

```diff
@@ -109,7 +109,7 @@ public void containsWordIgnoreCase_validInputs_correctResult() {
         assertFalse(StringUtil.containsWordIgnoreCase("    ", "123"));
 
         // Matches a partial word only
-        assertFalse(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bb")); // Sentence word bigger than query word
+        assertTrue(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bb")); // Sentence word bigger than query word
```

Comment:
[Question] why did you change this to true though? I thought this is for the method containsWordIgnoreCase and not equalsSubWordIgnoreCase. Did this pass the test? O.o 

Also, did you add test for equalsSubWordIgnoreCase?

Reply:
Regarding the first part, because originally, the containsWordIgnoreCase only works for the full word. Then I changed it to work for subword, so for "Rice", "ri" works as well, thus no longer assertFalse, since "aaa bbb ccc" contains "bb".

For the 2nd part, thanks! Will add it in.